libmamba/include/mamba/api/install.hpp

```diff
@@ -54,9 +56,9 @@ namespace mamba
 
         void create_empty_target(const fs::u8path& prefix);
 
-        void file_specs_hook(std::vector<std::string>& file_specs);
+        void file_specs_hook(std::vector<std::string>& file_specs, Configuration& config);
```

Comment:
Nitpicking: config should be the first argument to be consistent with other functions in this file.

Reply:
Ah yes nice catch, will fix